pkgs/development/python-modules/seaborn/default.nix

```
@@ -49,8 +49,16 @@ buildPythonPackage rec {
   ] ++ lib.optionals (!stdenv.hostPlatform.isx86) [
     # overly strict float tolerances
     "TestDendrogram"
+  ] ++ lib.optionals stdenv.isDarwin [
+    # Requires internet connection, and, when it's available without darwin
+    # sandbox, throws:
+    #   OSError: [Errno 30] Read-only file system: '/homeless-shelter'
```

```suggestion
    # Requires internet connection
```
The sandbox should always be enabled, __darwinAllowLocalNetworking is an exception for local networking, but this error can easily be fixed with ``export HOME=$(mktemp -d)``

---

From what it looked, the test requires the internet connection

---

Then we should still drop the second part of the comment because it suggests that the error is not easily fixable which it is.

---

Agreed